src/main/kotlin/org/dash/mobile/explore/sync/process/data/AtmData.kt

```diff
@@ -84,7 +84,7 @@ data class AtmLocation(
             setString(TERRITORY_COL, if (state.isNullOrEmpty()) city else state)
             setString(CITY_COL, city)
             setString(SOURCE_COL, "coinatmradar")
-            setInt(SOURCE_ID_COL, id)
+            setString(SOURCE_ID_COL, id.toString())
```

Comment:
Not sure about this change: the db schema still has this column as an `INTEGER`.

Reply:
I was looking at the wrong column.  `id` is an integer, `sourceId` is TEXT.

No issue after all.